athena-saphana/src/main/java/com/amazonaws/athena/connectors/saphana/SaphanaQueryStringBuilder.java

```diff
@@ -368,9 +380,25 @@ else if (singleValues.size() > 1) {
     }
 
     private String toPredicate(String columnName, String operator, Object value, ArrowType type,
-                               List<SaphanaQueryStringBuilder.TypeAndValue> accumulator)
+                               List<TypeAndValue> accumulator)
     {
-        accumulator.add(new SaphanaQueryStringBuilder.TypeAndValue(type, value));
+        accumulator.add(new TypeAndValue(type, value));
         return quote(columnName) + " " + operator + " ?";
     }
+
+    // copied from JdbcSplitQueryBuilder
+    private String extractOrderByClause(Constraints constraints)
```

Comment:
if this class extends JdbcSplitQueryBuilder, why are we copying methods between them? Can't you just change the visibility of `extractOrderByClause` in JdbcSplitQueryBuilder?

Reply:
resolved.